docs/events/timeline-ui-overview.asciidoc

```diff
@@ -117,26 +117,27 @@ Timeline, some fields may be disabled. For more information, see
 [[timeline-to-cases-ui]]
 == Attach Timeline to a case
 
-To attach a Timeline to a new or existing case, click *Attach to case* in the upper
+To attach a Timeline to a new or existing case, open it, click *Attach to case* in the upper
 right corner, and then select one of these:
 
-* *Attach timeline to new case*
-* *Attach timeline to existing case*
+* Attach to new case
+* Attach to existing case...
```

Comment:
I think these should be bold, because they are UI controls that we're telling the user to interact with.

Reply:
Agree with this. In the Security docs, I've only come across [one other instance](https://www.elastic.co/guide/en/security/master/timelines-ui.html) where menu options aren't bolded (see step 4), but this type of styling seems to be the exception rather than the rule.